sound/audioclass.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+from collections import deque
+import rospy
+import math
+import time
+import sys
+sys.path.append('../../ros-behavior-scripting')
+sys.path.append('../../../vision')
```

Comment:
will this work if run from a non-standard location? Perhaps there should be some sort of 'make install' step, so that the path is `/usr/local/lib/python3.4/site-packages` or something like that?  I think we also use `/usr/local/share/opencog/python`

I'm also sure that `../../../vision` is broken, I don't have this directory.  Also, on my system, my directory is called `ros-behave-git` and not `ros-behavior-scripting` so this is guaranteed to break, for me.